internal/scorecard/tests/bundle_test.go

```diff
@@ -276,15 +276,6 @@ var _ = Describe("Basic and OLM tests", func() {
 				result = checkOwnedCSVSpecDescriptors(cr, &csv, result)
 				Expect(result.State).To(Equal(scapiv1alpha3.FailState))
 			})
-			It("should fail when CRs do not have spec field specified", func() {
-				cr := []unstructured.Unstructured{
-					unstructured.Unstructured{
-						Object: map[string]interface{}{},
-					},
-				}
-				result = checkSpec(cr, result)
-				Expect(result.State).To(Equal(scapiv1alpha3.FailState))
-			})
```

Comment:
I think it would be more valuable to modify this test case to verify that the `SuccessState` is observed but a warning is issued. This way this code path is still covered by a test.

Reply:
Added the code and verified with `SuccessState`